### ISSUE TITLE
Port module plotting improvements from private repo

### ISSUE THREAD

**Issue by robertbartel** [NOAA-OWP/ngen-cal#101]
Short description explaining the high-level reason for the new issue.

## Steps to replicate behavior (include URLs)

1.

**Comment by aaraney** [comment]
I think this could be a good use case for a plugin once #126 is merged. I hesitate to just develop a plugin and put it in the source tree without it having been used. Instead, perhaps we can create an `examples` subdirectory in `python/ngen_cal` that has some example plugins? Or something like `python/ngen_cal/experimental_plugins`? What im getting at is, I think it makes more sense to build the idea and test / reshape it over time until we are happy with it and want to bring it in as a first class feature. Thoughts?
